cdap-proto/src/main/java/co/cask/cdap/proto/id/EntityId.java

```diff
@@ -85,10 +85,18 @@ protected EntityId(EntityType entity) {
 
   protected abstract Iterable<String> toIdParts();
 
+  /**
+   *  Deprecated method to rename to getEntityType(). Use {@link #getEntityType()}
+   */
+  @Deprecated
   public final EntityType getEntity() {
     return entity;
   }
 
+  public final EntityType getEntityType() {
+    return entity;
```

Comment:
I tried renaming it but there are a lot of places in docs that we'd have to rename the variable name too. I'll take another look.